TeadsSampleApp/Controllers/InRead/Direct/WebView/InReadDirectWebViewController.swift

```diff
@@ -24,11 +24,14 @@ class InReadDirectWebViewController: TeadsViewController, WKNavigationDelegate {
             return
         }
         let contentStringWithIntegrationType = contentString.replacingOccurrences(of: "{INTEGRATION_TYPE}", with: "InRead Direct WebView Integration")
+        
+        // The html identifier where you want your slot to open`
+        let htmlIdentifierForYourSlot = "#teads-placement-slot"
```

Comment:
naming is not correct, the selector is used in bootstrap.js [here](https://github.com/teads/TeadsSDK-iOS/blob/master/TeadsSampleApp/WebViewHelper/Resources/bootstrap.js#L280 ) to find slot using `querySelectorAll`, so I would suggest
```suggestion
        let domCSSSlotSelector = "#teads-placement-slot"
```